misk/src/main/kotlin/misk/web/actions/WebActionFactory.kt

```diff
@@ -40,11 +41,12 @@ internal class WebActionFactory @Inject constructor(
     webActionClass: KClass<A>,
     pathPrefix: String = "/"
   ): List<BoundAction<A>> {
-    // Find the function with Get, Post, or ConnectWebSocket annotation. Only one such function is
-    // allowed.
+    // Find the function with Get, Post, Delete or ConnectWebSocket annotation.
+    // Only one such function is allowed.
     val actionFunctions = webActionClass.functions.mapNotNull {
       if (it.findAnnotationWithOverrides<Get>() != null ||
```

Comment:
Wouldn't be a terrible idea to turn this into a loop over a list but a) I'll likely be adding support for other methods (PUT/PATCH) soon, and b) might as well make that change as its own PR.